tree/dataframe/src/RDFUtils.cxx

```diff
@@ -397,7 +397,7 @@ void CheckReaderTypeMatches(const std::type_info &colType, const std::type_info
    if (diffTypes && !inheritedType()) {
       const auto tName = TypeID2TypeName(requestedType);
       const auto colTypeName = TypeID2TypeName(colType);
-      std::string errMsg = where + ": type mismatch: column \"" + colName + "\" is being used as ";
```

Comment:
I saw that in the tests that `where` always is `tRVariationReader`. Could it be something else? Maybe we keep it `tRVariationReader` instead of `RDataFrame`. `RDataFrame` also looks good though.

Reply:
> Could it be something else?

yes, hence the change :smile: but indeed it looks like the only tests that check the error message are the ones in `dataframe_vary`